### PR TITLE
fix(Tabs): Add styling props to tabs wrapper

### PR DESCRIPTION
**Pull request by mayank99**
Without these props, users need to target our internal class names for layouts.

### CODE REVIEW

File: src/core/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -244,7 +248,11 @@ export const Tabs = (props: TabsProps) => {
   };
 
   return (
-    <div className={cx('iui-tabs-wrapper', `iui-${orientation}`)}>
+    <div
+      className={cx('iui-tabs-wrapper', `iui-${orientation}`, className)}
+      style={style}
+      id={id}
```
Comment (bentleyvk) [inline src/core/Tabs/Tabs.tsx:254]: Why not just `{...rest}`?

Comment (bentleyvk) [reply src/core/Tabs/Tabs.tsx:254]: Hm, then all `data-*` and `aria-*` attributes are passed not on the parent element.

Comment (mayank99) [reply src/core/Tabs/Tabs.tsx:254]: Yeah... this wrapper should have been a separate component

Comment (bentleyvk) [reply src/core/Tabs/Tabs.tsx:254]: Should it? Isn't it here because of selection animation?

Comment (mayank99) [reply src/core/Tabs/Tabs.tsx:254]: which one makes more sense?

```tsx
<Tabs {...billionProps} />
```
or

```tsx
<TabsWrapper>
  <Tabs />
  <TabPanel />
</TabsWrapper>
```

Comment (bentleyvk) [reply src/core/Tabs/Tabs.tsx:254]: When you write like this 😁 But it still would be `<Tabs {...sameProps} />`

Comment (mayank99) [reply src/core/Tabs/Tabs.tsx:254]: We would be able to get rid of these props.
image

And we would gain extra customization on all of those parts. Right now you can't do anything with the panel content other than apply class name.

**Comment by mayank99** [comment]
Closing this to reduce complexity. As a workaround, a flex wrapper can be used to control layouts.  In some cases (e.g. overflow) it might be necessary to apply `min-width: 0` but that can be solved using `> *`.

```tsx
<div className='custom-tabs-container'>
  <HorizontalTabs /> // or <VerticalTabs />
</div>
```

```scss
.custom-tabs-container {
  display: flex;
  // ...layout styles, e.g. height: 200px

  > * {
    min-width: 0;
  }
}
```

In v2 we should rethink the API and not forget to consider layouts.

**Pull request closed without merging**